rma/readme/CONTRIBUTORS.rst

```diff
@@ -7,3 +7,4 @@
 
 * Chafique Delli <chafique.delli@akretion.com>
 * Giovanni Serra - Ooops <giovanni@ooops404.com>
+* Michael Tietz (MT Software) <mtietz@mt-software.de>
```

Comment:
You should be also in the contributors for the adaptation work.

Reply:
I agree, but I am already added as a contributor.